samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/shoppingcarts/productitems/ProductItemsList.java

```diff
@@ -0,0 +1,49 @@
+package io.eventdriven.ecommerce.shoppingcarts.productitems;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public record ProductItemsList(
```

Comment:
The plural form is enough.
```suggestion
public record ProductItems
```